core/templates/dev/head/components/forms/FormBuilder.js

```diff
@@ -305,6 +305,8 @@ oppia.filter('isFloat', [function() {
   };
 }]);
 
+/* eslint-disable angular/directive-restrict */
```

Comment:
Remove newline below this one; remove newline above the corresponding enable statement. That makes it clearer that the pragma refers to this directive (by sandwiching it between the comments).

Ditto below.